ci/install.sh

```diff
@@ -19,7 +19,7 @@ ${CXX:-g++} -v
     fi
     make TH_VERSION="${TH_VERSION}"
 
-    make nkf.done moses.done mwerSegmenter.done pesq pyopenjtalk.done
+    make nkf.done moses.done mwerSegmenter.done pesq pyopenjtalk.done py3mmseg.done
```

Comment:
Why you need this in CI?

Reply:
These targets are installed to test the installation script. I'd like to test Makefile because sometimes it was broken due to my modification.
(Sometimes, pesq installation is failed due to the source server, so I'm thinking to delete pesq)

Reply:
ideally it should be in a different script like .github/workflows/make.yml
```yml
on:
  push:
    paths:
    - tools/Makefile

jobs:
  extra:
    steps:
    - run: make extra
```
to reduce the testing time

Reply:
Hmm, ideally, yes. More ideally, we should split integration test and unittest, and linter test  independently.
Okay, I keep it in mind, but not in this PR.

Reply:
OK. I'm not sure if codecov supports independent coverage xml files from different jobs.

Reply:
I think linter before testing is good early stopping. Anyway, the independent linter script is nice for local env.

Reply:
I see, it sounds hard, so we should keep integration tests and unit tests in one CI for now.